.github/workflows/ci.yaml

```diff
@@ -115,6 +115,23 @@ jobs:
       uses: googleapis/java-cloud-bom/tests/validate-bom@v26.13.0
       with:
         bom-path: gapic-libraries-bom/pom.xml
+  generation-config-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: validate generation configuration
+      shell: bash
+      run: |
+        docker run \
+          --rm \
+          -u "$(id -u):$(id -g)" \
+          -v "$(pwd):${workspace_name}" \
+          --entrypoint python \
+          gcr.io/cloud-devrel-public-resources/java-library-generation:"${library_generation_image_tag}" \
+          /src/cli/entry_point.py validate-generation-config
```

Comment:
In order to avoid specifying a custom entrypoint, it may be more convenient to have our docker image with `ENTRYPOINT [ 'python', '/src/cli/entry_point.py' ]` instead of `ENTRYPOINT [ 'python', '/src/cli/entrypoint.py', 'generate' ]. Just a nice to have